modules/javascript/web-worker/load.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Module Name: Web Worker
+ * Description: Run JavaScript in a separate Web Worker with the help of Partytown library.
+ * Experimental: Yes
+ *
+ * @since   1.0.0
+ * @package performance-lab
+ */
+
+/**
+ * PartyTown Configuration
+ *
+ * @since 1.0.0
+ * @see https://partytown.builder.io/configuration
+ * @return void
+ */
+function web_worker_partytown_configuration() {
+	$config = array(
+		'lib' => str_replace( site_url(), '', plugin_dir_url( __FILE__ ) ) . 'assets/js/partytown/',
+	);
+
+	/**
+	 * Add configuration for PartyTown.
+	 *
+	 * @since 1.0.0
+	 * @param array $config Configuration for PartyTown.
+	 * @return array
+	 */
+	$config = apply_filters( 'partytown_configuration', $config );
+
+	?>
+	<script>
+		window.partytown = <?php echo wp_json_encode( $config ); ?>;
+	</script>
+	<?php
+}
+add_action( 'wp_head', 'web_worker_partytown_configuration', 1 );
+
+/**
+ * Initialize PartyTown
+ *
+ * @since 1.0.0
+ * @return void
+ */
+function web_worker_partytown_init() {
+	wp_enqueue_script(
+		'partytown',
+		plugin_dir_url( __FILE__ ) . 'assets/js/partytown/partytown.js',
+		array(),
+		PERFLAB_VERSION,
+		false
+	);
+}
+add_action( 'wp_enqueue_scripts', 'web_worker_partytown_init', 1 );
+
+/**
+ * Get all scripts tags which has `partytown` dependency.
+ *
+ * @since 1.0.0
+ * @return void
+ */
+function web_worker_partytown_worker_scripts() {
+	global $wp_scripts;
+
+	$partytown_handles = array();
+
+	// Get all scripts which has `partytown` dependency.
+	foreach ( $wp_scripts->registered as $handle => $script ) {
+		if ( ! empty( $script->deps ) && in_array( 'partytown', $script->deps ) ) { // phpcs:ignore WordPress.PHP.StrictInArray.MissingTrueStrict
```

Comment:
why not use a strict comparison here?

Reply:
🤔 can't recall why but it seems there are no blockers to use strict comparison. So updated it to use strict comparison.